roles/network_plugin/calico/files/openssl.conf

```diff
@@ -0,0 +1,19 @@
+req_extensions = v3_req
+distinguished_name = req_distinguished_name
+
+[req_distinguished_name]
+
+[ v3_req ]
+basicConstraints = CA:FALSE
+keyUsage = digitalSignature, keyEncipherment
+
+[ ssl_client ]
+extendedKeyUsage = clientAuth, serverAuth
+basicConstraints = CA:FALSE
+subjectKeyIdentifier=hash
+authorityKeyIdentifier=keyid,issuer
+
+[ v3_ca ]
+basicConstraints = CA:TRUE
+keyUsage = digitalSignature, keyEncipherment, serverAuth
```

Comment:
`serverAuth` can appear in `extendedKeyUsage` not `keyUsage` , but `extendedKeyUsage` can be skipped for CA.
Also, `keyEncipherment` should not be set for CA. 
typical/recommended values: `cRLSign, keyCertSign [, digitalSignature]`
see https://www.openssl.org/docs/man1.1.0/man1/openssl-x509.html ,
https://www.openssl.org/docs/manmaster/man5/x509v3_config.html ,
https://tools.ietf.org/html/rfc5280#section-4.2.1.3 ,
https://tools.ietf.org/html/rfc5280#section-4.2.1.12